codepos not used in gcode_M28()

AnHardt opened the pull request:
Got:
Marlin_main.cpp:2567: warning: unused variable 'codepos'

Needs revision.
Don't now how this should look like.

AnHardt committed:
codepos not used in gcode_M28()

Got:
Marlin_main.cpp:2567: warning: unused variable 'codepos'

Needs revision.
Don't now how this should look like.

thinkyhead commented on Marlin/Marlin_main.cpp:
```diff
@@ -2564,7 +2564,7 @@ inline void gcode_M17() {
   inline void gcode_M28() {
-    char* codepos = strchr_pointer + 4;
+//    char* codepos = strchr_pointer + 4; // ??? not used ???
     char* starpos = strchr(strchr_pointer + 4, '*');
```
Could be used here... And on line 2574.

AnHardt committed:
Make use of codepos.

Saves one addition.
Needs one pointer on the stack.

Don't know if this is a go trade.
